talking-about-their-pronouns.md

```diff
@@ -0,0 +1,38 @@
+# Asking someone's pronouns
+
+## Over Slack / in conversation
+
+What: Sorry, just to check so I can get them right, do you mind me asking what your pronouns are?
+
+# Correcting someone who misgenders someone else
+
+## During a meeting
+
+### Respond in the moment
+
+When: Once the person who's just got the pronouns incorrect has finished talking
+How: Verbally
+What: Just to point out, I'm pretty sure their pronouns are they/them
+
+### Respond async
+
+Where: Slack DM, email etc.
+What: FYI, I think you used "she" to refer to Sarah and I'm pretty sure their pronouns are they/them :)
+
+## Via Slack/email
+
+Where: In a direct message to that person (Slack DM, email, i.e. not in the initial thread itself to avoid derailing conversation if it happened in a group conversation)
+What: FYI, I'm pretty sure Sarah uses they/them pronouns :)
+
+# When you get pronouns wrong
+
+## If you realise during the interaction
+
+When: In speech, not as a big deal
+How: Verbally
+What: Sorry, I just realised I've been getting your pronouns wrong.
+
+## If you realise later
+
+Where: In a direct message to that person (Slack DM, email etc.)
+What: Hey, I realised / got told I got your pronouns wrong earlier, sorry. I'm try and sometimes I mess up but I know it's important and I'll keep working at it. Feel free to point out if it seems I've forgotten.
```

Comment:
This might be far too long-winded?

Reply:
Similar to my above comment: if it was in the moment, I'd obviously apologise for getting it wrong - "they, sorry, he" - but I wouldn't dwell on it (unless I'd particularly fouled up, in which case we're not in script-territory any more, and you will have to exercise Full Personal Relationship skills!)

Reply:
I think some people would want to convey that they've noticed and are sorry, though that might be for their benefit rather than the person they've misgendered...

Maybe this needs qualifying with "It's probably enough just to get it right next time but you could always drop them a message along the lines of ..."?